pennylane_lightning/src/benchmarks/Bench_ApplyOperations.cpp

```diff
@@ -0,0 +1,196 @@
+#include <map>
+#include <random>
+#include <string>
+#include <string_view>
+#include <tuple>
+#include <utility>
+#include <vector>
+
+#include "Constant.hpp"
+#include "StateVectorManaged.hpp"
+
+#include <benchmark/benchmark.h>
+
+using OpName = std::string;
+using NumWires = size_t;
+using NumParams = size_t;
+
+using SerializedOp = std::tuple<OpName, NumWires, NumParams>;
+
+static inline auto getLightningGates()
+    -> std::map<OpName, std::pair<NumWires, NumParams>> {
```

Comment:
It probably from my code, but how about change this to 
```suggestion
    -> std::unordered_map<OpName, std::pair<NumWires, NumParams>> {
```